FAQ.md

```diff
@@ -25,40 +25,50 @@ Dragonfly makes it simple and cost-effective to set up, operate, and scale any
 
 No, Dragonfly can be used to distribute all kinds of files, not only container images. For downloading files by Dragonfly, please refer to [Download a File](https://github.com/dragonflyoss/Dragonfly/blob/master/docs/quick_start/README.md#downloading-a-file-with-dragonfly). For pulling images by Dragonfly, please refer to [Pull an Image](https://github.com/dragonflyoss/Dragonfly/blob/master/docs/quick_start/README.md#pulling-an-image-with-dragonfly).
 
-## What is SuperNode
+## What is the sequence of P2P distribution
 
-SuperNode is a long-time process with two primary responsibilities:
+Supernode will maintain a bitmap information about the correspondence between peers and pieces. When dfget really starts the download task, Supernode will return the info for several pieces（4 in default）for download according to the scheduler. **It depends on the scheduler to decide whether the download task will be dispatched to the supernode or other peers.** As for the details of the scheduler, please refer to [scheduler algorithm](#what-is-the-peer-scheduling-algorithm-in-default)
 
-- It's the tracker and scheduler in the P2P network that choose appropriate downloading net-path for each peer.
-- It's also a CDN server that caches downloaded data from source to avoid downloading same files repeatedly.
+## How supernode and peers manage cache files
```

Comment:
How supernode and peers manage cache files which is ready for other peer's pulling